### PR TITLE
Incremental training with layer freezing exp

### PR DESCRIPTION
**Pull request by rnyak**
This PR cleans and organizes the example [09-Incremental-Training-with-Different-Learning-Rates-and-Layer-Freezing.ipynb](https://github.com/NVIDIA-Merlin/models/pull/667/files/1165d07b6765471dc7cb697918068f93184db873#diff-cb7137ba6d70ba60f4e5bfa06cf5d5121667447eb7ca41f339c1329e02203af2) in this PR  https://github.com/NVIDIA-Merlin/models/pull/667.

Missing: Will update the logo.

### CODE REVIEW

**Review by bschifferer — CHANGES_REQUESTED** [review]
It does not contain a unittest

**Review by bschifferer — CHANGES_REQUESTED** [review]
NVIDIA Logo is not customized

File: examples/usecases/incremental-training-with-layer-freezing.ipynb [hunk 1]
```diff
@@ -0,0 +1,883 @@
+{
```
Comment (bschifferer) [inline examples/usecases/incremental-training-with-layer-freezing.ipynb:1]: <p>Can we add more descrption to</p><p> </p><p>"In this example, we fine-tune a model by setting different learning rates to different layers and freezing embedding tables."</p><p> </p><p>Why do we want to use different learning rates? Why do we freeze the embedding tables?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NVIDIA-Merlin/models/pull/809/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Comment (bschifferer) [inline examples/usecases/incremental-training-with-layer-freezing.ipynb:1]: <p>Should we rename the dataset to day_1, day_2 and day_3 (and add that this is hypothetical)?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NVIDIA-Merlin/models/pull/809/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Comment (bschifferer) [inline examples/usecases/incremental-training-with-layer-freezing.ipynb:1]: <p>I think we should change the headers to fit more the different objectives.</p><p> </p><p>Building a Tow Tower Model</p><p>- Iteration 1: Using different Learning Rates</p><p>-- Training on First Day's Data</p><p>-- Training on Second Day's Data</p><p>- Iteration 2: Training with Freezing Layers</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NVIDIA-Merlin/models/pull/809/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>

Comment (bschifferer) [inline examples/usecases/incremental-training-with-layer-freezing.ipynb:1]: <p>Let's add a statement that the data is random and therefore the evaluation metrics will not "make sense"</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NVIDIA-Merlin/models/pull/809/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>

Comment (bschifferer) [inline examples/usecases/incremental-training-with-layer-freezing.ipynb:1]: <p>I would write it clear, that this is a new scenario. </p><p> </p><p>"Let's consider a new situation..." it is unrelated to the situation above and we train on data_1 again?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NVIDIA-Merlin/models/pull/809/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='22'/>

**Comment by bschifferer** [comment]
it looks good, but we still miss a unittest

**Comment by bschifferer** [comment]
we still need to update the logo

**Review by bschifferer — APPROVED** [review]

**Comment by bschifferer** [comment]
rerun tests

**Comment by rnyak** [comment]
rerun tests

**Comment by rnyak** [comment]
rerun tests

**Comment by rnyak** [comment]
rerun tests

**Pull request merged**